web/onos-gui/src/app/onos-topo/device-detail/device-detail.component.css

```diff
@@ -31,18 +31,22 @@
  */
 
 
-#device-detail-panel.floatpanel {
+ #device-detail-panel.floatpanel {
     z-index: 0;
     font-size: 10pt;
     top: 185px;
-    height: 400px;
-    overflow-y: scroll;
+    height: 50vh;
+    overflow-y: auto;
 }
 
 #device-detail-panel.floatpanel a {
     font-weight: bold;
 }
 
+#device-detail-panel .middle {
+    height: 40vh;
```

Comment:
I think change this to 365px, just so that the end of it reaches the end of the overall "floatpanel"

also add the the "overflow-y: auto" to this

Reply:
done